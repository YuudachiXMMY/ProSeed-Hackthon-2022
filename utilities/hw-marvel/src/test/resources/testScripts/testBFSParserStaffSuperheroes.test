# Test Marvel BFS and parser for an alphabetically least path

LoadGraph staffHeroes staffSuperheroes.tsv

FindPath staffHeroes Notkin-of-the-Superhuman-Beard Grossman-the-Youngest-of-them-all