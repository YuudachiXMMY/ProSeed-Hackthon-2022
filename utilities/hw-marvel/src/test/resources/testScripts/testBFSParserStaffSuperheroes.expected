# Test Marvel BFS and parser for an alphabetically least path

loaded graph staffHeroes

path from Notkin-of-the-Superhuman-Beard to Grossman-the-Youngest-of-them-all:
Notkin-of-the-Superhuman-Beard to Grossman-the-Youngest-of-them-all via CSE331